ambry-quota/src/main/java/com/github/ambry/quota/AmbryQuotaManager.java

```diff
@@ -23,6 +23,7 @@
 import com.github.ambry.messageformat.BlobInfo;
 import com.github.ambry.rest.RestRequest;
 import com.github.ambry.utils.Utils;
+import java.util.ArrayList;
```

Comment:
Note that new QuotaManager changes are out of scope for this PR. The only changes made to QuotaManager is to enable the current QuotaManager to continue working unchanged. Changes in QuotaManager will come in subsequent PRs.
The scope of this PR is to introduce the new QuotaEnforcer and QuotaSource for CapacityUnit quota, and make changes in other code so that the overall behavior remains unchanged.